lib/pennyworth/commands/import_base_command.rb

```diff
@@ -92,10 +104,14 @@ def fetch_remote_box_state_file
     end
 
     def base_image_clean(box)
-      Cheetah.run "virsh", "-c", "qemu:///system", "vol-delete", "#{box}_vagrant_box_image.img", "--pool=default"
-      Cheetah.run "virsh", "-c", "qemu:///system", "pool-refresh", "--pool=default"
-      @vagrant.destroy
-    rescue
+      image = get_virsh_image_name(box)
+      if !image.empty?
+        Cheetah.run(
+          "virsh", "-c", "qemu:///system", "vol-delete", image, "--pool=default"
+        )
+        Cheetah.run "virsh", "-c", "qemu:///system", "pool-refresh", "--pool=default"
+        @vagrant.destroy
+      end
```

Comment:
No, since import images should work without errors if there was no image imported before.